src/WebSocketsServiceProvider.php

```diff
@@ -56,19 +57,19 @@ public function boot()
 
     protected function configurePubSub()
     {
-        if (config('websockets.replication.enabled') !== true || config('websockets.replication.driver') !== 'redis') {
+        if (config('websockets.replication.driver') === 'local') {
```

Comment:
This is technically a breaking change - just a reminder to make sure to note that in the v2 release notes that the replication config will need to be added when upgrading. Or you could keep BC by just defaulting to local if no driver is set.

Reply:
Nice catch 🐛